core/src/test/java/org/elasticsearch/search/aggregations/InternalAggregationTestCase.java

```diff
@@ -122,4 +124,16 @@ protected final T createUnmappedInstance(String name) {
     protected NamedWriteableRegistry getNamedWriteableRegistry() {
         return namedWriteableRegistry;
     }
+
+    /**
+     * @return a random {@link DocValueFormat} that can be used in aggregations which
+     * compute numbers.
+     */
+    protected static DocValueFormat randomNumericDocValueFormat() {
+        final List<Supplier<DocValueFormat>> formats = new ArrayList<>(3);
+        formats.add(() -> DocValueFormat.RAW);
+        formats.add(() -> DocValueFormat.BOOLEAN);
```

Comment:
In which case I would say remove the BOOLEAN format here since its not actually a numeric format